android/src/main/java/com/reactnativestripesdk/CardFormView.kt

```diff
@@ -159,9 +166,17 @@ class CardFormView(context: ThemedReactContext) : FrameLayout(context) {
       }
     }
     fontFamily?.let {
+      // Load custom font from assets, and fallback to default system font
+      val typeface = ReactTypefaceUtils.applyStyles(null, -1, -1, it.takeIf { it.isNotEmpty() }, context.assets)
       for (binding in editTextBindings) {
-        binding.typeface = Typeface.create(it, Typeface.NORMAL)
+        binding.typeface = typeface
+      }
+      for (binding in inputLayoutBindings) {
```

Comment:
```suggestion
      for (binding in placeholderTextBindings) {
```

see suggestion above for context

Reply:
Changed.